students/yoshitsugunagao/styles.css

```diff
@@ -0,0 +1,98 @@
+body {
+  margin: 0;
+  padding: 0;
+  /* background-color: powderblue; */
+  font-family: 'Montserrat', sans-serif;
+}
+header{
+  background-color:#343434;
+  color:white;
+  text-align: center;
+}
+
+.top{
+  padding-bottom: 20px;
+}
+
+header h1{
+  font-family: 'Montserrat', sans-serif;
+  padding-top: 10px;
+  margin: 0;
+  font-size: 2rem;
+}
+
+header p{
+  text-transform: capitalize;
+  font-size:1.5rem;
+}
+
+.profile-photo{
+  width: 80%;
+  max-width: 300px;
+}
+
+.section-container{
+
+  margin: 0 10%;
+}
+
+
+.about-me-section{
+
+}
+
+.exercises-section{
+
+}
+
+
+
```

Comment:
Add a comment saying this is the css for the comment section for example, or something like that. Css files can get very long and messy if it is not structured. Will do the merge after that ;)